packages/functions/src/notifications/NotificationSender.ts

```diff
@@ -52,8 +56,12 @@ export async function sendEmailToDonor(
     templateId: templateId,
     dynamicTemplateData: data,
   };
-
-  addCalendarEventToDonor(msg, type, data, donor.email);
+  if (
+    type == NotificationToDonor.APPOINTMENT_BOOKED ||
+    type == NotificationToDonor.APPOINTMENT_CANCELLED_BY_COORDINATOR
```

Comment:
I think this should be a switch/case, in an external function.

i.e. `shouldAddCalendarEvent(type)`